targets/PythonSdk/make.js

```diff
@@ -1,10 +1,27 @@
 var path = require("path");
+var fs = require("fs");
 
 // Making resharper less noisy - These are defined in Generate.js
 if (typeof (templatizeTree) === "undefined") templatizeTree = function () { };
 if (typeof (getCompiledTemplate) === "undefined") getCompiledTemplate = function () { };
 
+function copyOverScripts(sourceDir, apiOutputDir)
+{
+    var srcDir = sourceDir + "\\source";
+    var outDir = apiOutputDir.substr(0, 18);
+    console.log("copying scripts from: " + srcDir + " to: " + apiOutputDir);
+
+    var filesInDir = fs.readdirSync(srcDir);
```

Comment:
You don't have to do this. Use templatizeTree(), which is defined in generate.js, and automatically available global function.